README.md

```diff
@@ -1,4 +1,6 @@
-# Capstone project
+# Capstone project 
+
+Loom link: [https://www.loom.com/share/f1b2f54ec37b4f7f859063a722de7b26]
 
 Original design idea by Cindy Shin in Behance.[https://www.behance.net/adagio07]
 
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md) for this project(`Check under Day 3`), " `Deploy the project and check to make sure everything works as planned` ", kindly deploy your site using `GitHub Pages` and add the link in your readme file. A link to the deployed project should be present in your README